deploy/sre-prometheus/100-rbac-permission-operator.PrometheusRule.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-rbac-permission-operator
+    role: alert-rules
+  name: sre-rbac-permission-operator
+  namespace: openshift-monitoring
```

Comment:
This is a bikeshed question, but any reason not to put this in the rbac-permissions-operator namespace?

Reply:
I'm not sure.  Let me check with the monitoring folks to see if they can be re-homed.

Reply:
https://github.com/openshift/rbac-permissions-operator/pull/16 sets up for the move, and 582f32e handles the move.